nautobot_chatops/workers/helper_functions.py

```diff
@@ -0,0 +1,98 @@
+"""Helper functions for nautobot.py worker."""
+
+
+NETBOX_LOGO_PATH = "nautobot/NautobotLogoSquare.png"
+NETBOX_LOGO_ALT = "Nautobot Logo"
+
+
+def nautobot_logo(dispatcher):
+    """Construct an image_element containing the locally hosted Nautobot logo."""
+    return dispatcher.image_element(dispatcher.static_url(NETBOX_LOGO_PATH), alt_text=NETBOX_LOGO_ALT)
+
+
+def menu_item_check(item):
+    """Return True if value starts with 'menu_offset' for dealing with Slack menu display limit."""
+    if item:
+        try:
+            return item.startswith("menu_offset-")
+        except AttributeError:
+            return False
+    return True
```

Comment:
Logic seems wrong here to me:

```python
>>> menu_item_check("menu_offset-1")
True
>>> menu_item_check("hello")
False
>>> menu_item_check("")
True
>>> menu_item_check(None)
True
```

Shouldn't we be returning `False` for an empty string or `None`?

Reply:
> if not filter_value_1 or is_menu_track_item(filter_value_1):
>
> A general nitpick - I find logic of the form if not x or y to be a bit non-obvious. Would you be able to change these throughout to if y or not x (such as if is_menu_track_item(filter_value_1) or not filter_value_1) and still have things work as intended?

This was created specifically to address the above comment, and condense the logic into a single function.